@uportal/esco-content-menu/README.md

```diff
@@ -217,6 +237,7 @@ Standalone properties:
 - `hide-action: type`: `Boolean`, default: `false`, define to hide or not the `action-favorite` button defined into `portlet-card`
 - `show-footer-categories`: `Boolean`, default: `false`, define to display category dropdown filter near bottom of grid
 - `hide-title`: `Boolean`, default: `false`, define to remove the title area from the grid, useful when a basic grid is desired
+- `portlet-background-is-dark`: `Boolean`, default: `false`, indicate to the component that the parent background is dark and permit to the portlet-card component to show buttons like favorites in a more suitable color.
```

Comment:
Would this be better to make this a css variable?
Allowing more fine grained control of the contrasting color.

Reply:
I don't know as it could be controlled from different points, as example with the esco-content-menu the content-grid used to show all services need this property as false whereas the favorite-content to true at the same time, so how to provide and use 2 values in this case ?
After instead of providing a boolean as prop I can use 2 props each one for a dark and light color, like that each peoples will be able to customize it.

Reply:
I was thinking there would be a foreground and a background color.

Reply:
@jgribonvald having two css variables, for foreground and background would be preferable.
A Boolean won't be able to handle all the possible cases for keeping contrast high.

Reply:
@ChristianMurphy as already told on slack this boolean property is not to add a feature, it's only to add a missing control that is required for all improvements around, and more this boolean is already used on other module. To move all this stuff on css variable there will need to rewrite on different components. So it's an other work and so an other PR.

So in my mind this can be merged.